src/modules/ROOT/pages/changelog/202209/07.json

```diff
@@ -0,0 +1,179 @@
+{
+  "meta": {
+    "releasedOn": "2023.02.10"
+  },
+  "added": [
+    {
+      "what": [
+        "support for casting numbers to float inside workflow expressions via the float function."
+      ],
+      "why": [
+        "to simplify type casting by using a function instead of an intermediate workflow expression variable."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/38b6cb35e703377cce14b507be601e176face8a2"
+      ]
+    },
+    {
+      "what": [
+        "ability to toggle block border."
+      ],
+      "why": [
+        "to make better distinction between blocks when they are not shown as a cards."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/860/commits/6308bdfebb0b22cc5073de817cf7ba233690f213"
+      ]
+    },
+    {
+      "what": [
+        "general UI/UX improvements."
+      ],
+      "why": [
+        "to make some invalid configurations impossible, better overview of some resources and prepare styles and layouts for future releases."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/80117c09f099ff8806a3608ba8548abf80b3c726",
+        "https://github.com/cortezaproject/corteza/commit/ef7e8649a5e97f7a14988d037f08351eefa6f986",
+        "https://github.com/cortezaproject/corteza/commit/9acc1099279b2f1e4aadee9f46d2315a919a2feb"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "making name property required when creating templates, dal connections, workflows, auth clients and sensitivity levels."
```

Comment:
`making name property...` -> `making the name property...`

`...auth clients and sensitivity...` -> `...auth clients, and sensitivity...`